src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,102 @@
+.icon {
+  display: block;
+  width: 100%;
+  height: 18px;
+
+  @include onTablet {
+    height: 24px;
+  }
+
+  &--phone {
+    @extend %bgImageNormalize;
+    background-image: url(../images/met_phone_icon.svg);
+
+    @include onDesktop {
+
+      &::before {
+        content: "The Met Fifth Avenue +1 212-535-7710";
```

Comment:
Move hard-coded text 'The Met Fifth Avenue +1 212-535-7710' into a separate variable or a data attribute to make it easier to manage and update.